mysql-test/suite/innodb/t/temporary_table.test

```diff
@@ -633,7 +633,7 @@ drop table t1;
 #
```

Comment:
What is the path of this file in 5.7 HEAD?

Reply:
This is a new 8.0 file but the problem is the same `sync_latch_meta_init()` memory leak.